amostra/schemas/sample.json

```diff
@@ -12,16 +12,13 @@
             "type": "string",
             "description": "Sample name"
         },
-        "projects": {
```

Comment:
I think we want to keep this field, but instead of being a list of foreign keys to documents/objects in amostra, it will be a lit of references to _external_ systems such as PASS. Thus, for the purposes of this PR, projects is any string. (We might eventually want to restrict that it should be a string formatted like as URL, as in `scheme://path`.